app/planets_routes.py

```diff
@@ -0,0 +1,138 @@
+from app import db
+from app.models.planet import Planet
+from app.models.moon import Moon
+from flask import Blueprint, jsonify, abort, make_response, request
+from sqlalchemy import desc, asc
+
+
+# ---------helper functions--------
+def sort_planet_query(planet_query, sort_query, order_query):
+    if not sort_query:
+        sort_query = "name"
+
+    if not order_query:
+        order_query = "asc"
+
+    if order_query == "desc":
+        planet_query = planet_query.order_by(desc(sort_query))
+    else:
+        planet_query = planet_query.order_by(asc(sort_query))
+
+    return planet_query
+
+
+
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response(jsonify({"message":f"{cls.__name__} {model_id} invalid"}), 400))
+
+    model = cls.query.get(model_id)
+
+    if not model:
+        abort(make_response(jsonify({"message":f"{cls.__name__} #{model_id} not found"}), 404))
```

Comment:
Around best practices, there are many lines across the files that break the PEP 8 guide of 79 characters max per line. I would keep an eye out for this in the future, especially on lines where there are nested function calls.